src/in-depth/human-communication.md

```diff
@@ -16,18 +16,60 @@ Style of writing:
 
 ## When it's hard to tell what's going on
 
-<aside class="todo">
+When communicating non-nominal state it's important to be consistent.
+A heavily logging application that doesn't follow strict logging levels
+provides the same amount, or even less information
+than a non-logging application.
+
+Because of this it's important to define the severity of events
+and messages that are related to it;
+then use consistent log levels for them.
+This way users can select the amount of logging themselves
+via `--verbose` flags.
+
+Available levels should be
```

Comment:
log has trace but not fatal: https://docs.rs/log/0.4.4/log/enum.Level.html (feel free to also link to these docs!)